README.md

```diff
@@ -176,36 +176,82 @@ Parameters:
 - type: String
 - example: 659bbda98db7a017ddd9147a
 - description: ADP application id to be updated. It can be obtained from ADP app version page- it is available in url
-4. imagename
+4. appVersion
```

Comment:
can we not require this parameter? As an app developer, I may only want to know what the version just created is, without need to provide any version.

Reply:
readme updated for removal of extra jenkins pipeline